.vib/matomo/cypress/cypress/integration/matomo_spec.js

```diff
@@ -1,74 +1,62 @@
 /// <reference types="cypress" />
 import { random } from '../support/utils';
 
-it('allows to log in and out', () => {
-  cy.login();
-  cy.get('a[href*="logout"]').first().click();
-  cy.contains('Sign in');
-});
-
-it('allows to create user', () => {
-  cy.login();
-  cy.visit('/?module=UsersManager&showadduser=1');
-  cy.fixture('users').then((user) => {
-    cy.get('#user_login').type(`${user.newUser.username}-${random}`);
-    cy.get('#user_password').type(`${user.newUser.password}-${random}`);
-    cy.get('#user_password').type(`${user.newUser.password}-${random}`);
-    cy.get('#user_email').type(`${random}_${user.newUser.email}`);
-    cy.get('input[value*="Create user"]').click();
-    cy.contains('changes have been saved');
-    cy.visit('/index.php?module=UsersManager');
-    cy.contains('#userLogin', `${user.newUser.username}-${random}`);
-  });
-});
-
 it('allows to create a new website', () => {
```

Comment:
I am missing a lot of context about the asset itself, but does this test verify anything that the following one does not? In the same vein, are any of these tests checking the matomo PVC works as expected?

Reply:
>  but does this test verify anything that the following one does not?
Yes, the original intention was to test two different components of the app: the website itself and the API.

> are any of these tests checking the matomo PVC works as expected?
About the PVC, they don't perform any changes in the fs (I suppose they are stored in the DB). Nevertheless, based on the folder structure, I don't think there are many operations that modify it.

```
$ ls /bitnami/matomo
config	js  misc  plugins
```